doc/source/user-experience.rst

```diff
@@ -57,6 +57,8 @@ existing image, such as the ``scipy-notebook`` image, complete these steps:
            tag: 8e15d329f1e9
 
    .. note::
+   
+      Container name cannot be longer than 63 characters.
```

Comment:
This should probably say that the container *image* name can not be longer than 63 characters?

Reply:
Indeed, thanks. Fixed.